build/IceRpc.Examples.props

```diff
@@ -1,9 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <Import Project="$(MSBuildThisFileDirectory)IceRpc.Version.props" />
     <PropertyGroup>
+        <!-- The version of the IceRPC NuGet packages used by the examples. It must be available from nuget.org. -->
```

Comment:
Need to update the comment.

Maybe something like:
It must be available from nuget.org once this branch is released.